libflux/flux-core/src/semantic/doc.rs

```diff
@@ -559,6 +561,34 @@ fn comments_to_string(comments: &[ast::Comment]) -> String {
     s
 }
 
+/// Shorten removes all long form descriptions from the docs structures leaving only the headlines
+/// and other metadata.
```

Comment:
Super disappointed we didn't find a way to add "if you're not into that whole brevity thing" into the docs here. 🤪😂

Reply:
Had to look it up :D